tianracer_gazebo/launch/demo_tianracer_teb_nav.launch

```diff
@@ -1,28 +1,38 @@
 <launch>
     <!-- Config map file -->
     <arg name="world" default="tianracer_racetrack" />
+    <arg name="robot_name" default="tianracer_01" />
```

Comment:
https://github.com/tianbot/tianbot/blob/master/tianbot_bringup/launch/base.launch

Please refer to the tianbot base.launch for the robot_name assignment